Runner.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,15 @@
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Runner.Tests
+{
+    public class ProgramTests
+    {
+        [Test]
+        public async Task Main_StartupWorks()
+        {
+            // This is just a quick dumb test for DI
+            await Program.Main(new string[0]);
+        }
+    }
+}
```

Comment:
No Newline at EOF

Reply:
Good catch, I added this setting to .vscode-settings so it shouldn't be an issue anymore.